Common/yaml/Contacts.yaml

```diff
@@ -0,0 +1,374 @@
+openapi: 3.0.3
+info:
+  title: Smart Contact Mangaer OpenAPI 3.0
+  description: The Project was about creating smart contact manager.Which can be specifically used for business model.This is a web application which can be used as a simple pluging to any existing project.|-
+   
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: kondangkelbablu1998@gmail.com
+  license:
+    name: Smart Contact Manager
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.11
+externalDocs:
+  description: Find out more about Smart Contact Manager
+  url: http://swagger.io
+servers:
+  - url: http://localhost:8082
+tags:
+  - name: contacts
+    description: Apis related to contacts
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+  - name: categories
+    description: Apis related to categories
+    externalDocs:
+      description: Find out more about our store
+      url: http://swagger.io
+  - name: user
+    description: Operations about user
+paths:
+  /v1/contacts:
+    post:
+      tags:
+        - contacts
```

Comment:
Change the tag here and other paths as per the previous comment

Reply:
fixed please review